### DISCUSSION TITLE
Reading OpenTTD files with rust

### DISCUSSION THREAD

**Discussion by fernandobatels** [OpenTTD/OpenTTD#11796] [General]
Just sharing a small project with rust for read the save files: https://github.com/fernandobatels/ropenttd

**Comment by TrueBrain** [comment]
Nice!

If you ever need inspiration, a savegame visualiser can be found here:

https://github.com/TrueBrain/OpenTTD-savegame-reader

It also contains a bit of Rust code how to read savegames (as it uses WASM to make it happen). The result is published here: https://truebrain.github.io/OpenTTD-savegame-reader/

Enjoy fiddling with our savegames; they contain a lot of cool information :D
